### PR TITLE
Add troubleshooting section about native change warnings

### PR DESCRIPTION
**Pull request by bryanoltman**
## Status

**READY**

## Description

Part of https://github.com/shorebirdtech/shorebird/issues/1286. https://github.com/shorebirdtech/shorebird/issues/1286 will be closed once we add a link to this in the CLI.

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -122,6 +122,29 @@ version and build numbers in order to target patches at specific releases. You
...
+   use caution when publishing patches that include changes to native code from
+   plugins. In the worst case, these changes my cause your patched app to
+   crash.**
+2. A dependency/plugin produces a different output on every build. This can
+   happen if the dependency it includes a timestamp indicating when it was
+   built, for example. This kind of change is usually safe to publish, but you
+   should be sure this is the only reason you are seeing this warning.
+
+If you are confident that the changes are safe, you can ignore this warning by
+passing the `--force` flag to `shorebird patch`.
```
Comment (bryanoltman) [inline docs/troubleshooting.md:146]: Maybe we should add an `--ignore-native-changes` or `--ignore-change-warning` flag instead?

Comment (felangel) [reply docs/troubleshooting.md:146]: In addition to force?

**Review by felangel — APPROVED** [review]

**Comment by bryanoltman** [comment]
@felangel Yeah. For the purposes of artifact diffing, it would behave as though the `--force` flag were provided, but would not skip any other checks. I'm mostly suggesting it because I don't think we want to encourage users to use `--force` in more or less any case.

**Comment by vivaan-vama** [comment]
This have been publish to production ?

**Pull request merged**